specifications/api/swagger.yaml

```diff
@@ -845,6 +853,20 @@ definitions:
       passphrase: *walletPassphrase
       address_pool_gap: *walletAddressPoolGap
 
+  ApiAccountPostData: &ApiAccountPostData
+    type: object
+    required:
+      - name
+      - account_public_key
+    properties:
+      name: *walletName
+      account_public_key: *walletAccountPubkey
+      address_pool_gap: *walletAddressPoolGap
+
+  ApiWalletOrAccountPostData: &ApiWalletOrAccountPostData
```

Comment:
so I have problem here. how to express `oneof` . This one unfortunately gives : 
```
 1) Cardano.Wallet.Api.Types, verify that every type used with JSON content type in a servant API has compatible ToJSON and ToSchema instances using validateToJSON., WalletOrAccountPostData
       Falsifiable (after 3 tests):
         WalletOrAccountPostData {postData = Right (AccountPostData {name = ApiT {getApiT = WalletName {getWalletName = ":p6&vFta~? Yp\147454n\30864rWs~\"!r#:ms 3+KZUelcepj>n$0F99`$UVGunr\27692}V0xnw3+0CJ_fe/\152955N|)EBVO6Yd[u%,(rSJ:"}}, accountPublicKey = AccountPublicKey {key = ApiT {getApiT = ShelleyKey {getKey = XPub {xpubPublicKey = "Qd\237Bm\207\154\179\ESC>\208I\219\236c|\b\174\141\199w\179\SUB\242\183\RS\219\186\&7\179\&5\194", xpubChaincode = ChainCode "\226c\ENQP\DC3#\FS\"\221\133\150n@[33\154\138xh\133%\187V\181^6,v\220\131l"}}}}, addressPoolGap = Nothing})}
         Validation against the schema fails:
           * property "mnemonic_sentence" is required, but not found in "{\"account_public_key\":\"5164ed426dcf9ab31b3ed049dbec637c08ae8dc777b31af2b71edbba37b335c2e263055013231c22dd85966e405b33339a8a78688525bb56b55e362c76dc836c\",\"name\":\":p6&vFta~? Yp\240\163\191\190n\231\162\144rWs~\\\"!r#:ms 3+KZUelcepj>n$0F99`$UVGunr\230\176\172}V0xnw3+0CJ_fe/\240\165\149\187N|)EBVO6Yd[u%,(rSJ:\"}"
           * property "passphrase" is required, but not found in "{\"account_public_key\":\"5164ed426dcf9ab31b3ed049dbec637c08ae8dc777b31af2b71edbba37b335c2e263055013231c22dd85966e405b33339a8a78688525bb56b55e362c76dc836c\",\"name\":\":p6&vFta~? Yp\240\163\191\190n\231\162\144rWs~\\\"!r#:ms 3+KZUelcepj>n$0F99`$UVGunr\230\176\172}V0xnw3+0CJ_fe/\240\165\149\187N|)EBVO6Yd[u%,(rSJ:\"}"
           * property "account_public_key" is found in JSON value, but it is not mentioned in Swagger schema
         
         JSON value:
         {
             "account_public_key": "5164ed426dcf9ab31b3ed049dbec637c08ae8dc777b31af2b71edbba37b335c2e263055013231c22dd85966e405b33339a8a78688525bb56b55e362c76dc836c",
             "name": ":p6&vFta~? Yp𣿾n碐rWs~\"!r#:ms 3+KZUelcepj>n$0F99`$UVGunr氬}V0xnw3+0CJ_fe/𥕻N|)EBVO6Yd[u%,(rSJ:"
         }
         
         Swagger Schema:
         {
             "required": [
                 "name",
                 "mnemonic_sentence",
                 "passphrase"
             ],

```

Reply:
fixed in https://github.com/input-output-hk/cardano-wallet/pull/1360/commits/794224a42c9a7f7e646bdd58a76572f670c1260c